pkg/toolsets/kubevirt/vm/lifecycle/tool.go

```diff
@@ -0,0 +1,140 @@
+package lifecycle
+
+import (
+	"fmt"
+
+	"github.com/containers/kubernetes-mcp-server/pkg/api"
+	"github.com/containers/kubernetes-mcp-server/pkg/kubevirt"
+	"github.com/containers/kubernetes-mcp-server/pkg/output"
+	"github.com/google/jsonschema-go/jsonschema"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/utils/ptr"
+)
+
+// Action represents the lifecycle action to perform on a VM
+type Action string
+
+const (
+	ActionStart   Action = "start"
+	ActionStop    Action = "stop"
+	ActionRestart Action = "restart"
+)
+
+func Tools() []api.ServerTool {
+	return []api.ServerTool{
+		{
+			Tool: api.Tool{
+				Name:        "vm_lifecycle",
+				Description: "Manage VirtualMachine lifecycle: start, stop, or restart a VM",
+				InputSchema: &jsonschema.Schema{
+					Type: "object",
+					Properties: map[string]*jsonschema.Schema{
+						"namespace": {
+							Type:        "string",
+							Description: "The namespace of the virtual machine",
+						},
+						"name": {
+							Type:        "string",
+							Description: "The name of the virtual machine",
+						},
+						"action": {
+							Type:        "string",
+							Enum:        []any{"start", "stop", "restart"},
+							Description: "The lifecycle action to perform: 'start' (changes runStrategy to Always), 'stop' (changes runStrategy to Halted), or 'restart' (stops then starts the VM)",
+						},
+					},
+					Required: []string{"namespace", "name", "action"},
+				},
+				Annotations: api.ToolAnnotations{
+					Title:           "Virtual Machine: Lifecycle",
+					ReadOnlyHint:    ptr.To(false),
+					DestructiveHint: ptr.To(false),
```

Comment:
```suggestion
					DestructiveHint: ptr.To(true), // stop/restart can disrupt workloads
```
IMO this annotation is wrong, as stopping or restarting a VM can disrupt workloads